pkgs/applications/graphics/megapixels/default.nix

```diff
@@ -26,12 +26,12 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "megapixels";
-  version = "0.15.0";
+  version = "0.16.0";
 
   src = fetchgit {
```

Comment:
```suggestion
  src = fetchFromSourcehut {
```

Reply:
Oh cool, didn't know such a fetcher was added. Will adjust it.

Reply:
Switched to `fetchFromSourcehut` fetcher.